Assets/Scripts/SS3D/Data/AssetDatabases/InspectorEditor/AssetDatabaseInspectorEditor.cs

```diff
@@ -0,0 +1,92 @@
+﻿#if UNITY_EDITOR
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using SS3D.CodeGeneration;
+using UnityEditor;
+using UnityEditor.AddressableAssets.Settings;
+using UnityEditor.UIElements;
+using UnityEngine;
+using UnityEngine.UIElements;
+using Object = UnityEngine.Object;
+
+namespace SS3D.Data.AssetDatabases.InspectorEditor
+{
+    [CustomEditor(typeof(AssetDatabase))]
+    public class AssetDatabaseInspectorEditor : Editor
+    {
+        private AssetDatabase _assetDatabase;
+
+        public VisualTreeAsset _assetDatabaseVisualTree;
+
+        private ScrollView _assetsListView;
+        private Button _loadAssetsButton;
+        private ObjectField _assetGroupObjectField;
+        private Label _assetDatabaseLabel;
+        private TextField _enumNameTextField;
+
+        private static readonly Regex SlashRegex = new(@"[\\//]");
+
+        private void OnEnable()
+        {
+            _assetDatabase = (AssetDatabase)target;
+        }
+
+        public override VisualElement CreateInspectorGUI()
+        {
+            VisualElement root = new();
```

Comment:
Make a comment to explain what is expected to be displayed in inspector with this